domains/mail.letum.json

```
@@ -0,0 +1,14 @@
+{
+   "owner": {
+      "username": "amoramicae",
+      "discord": "1252631745514180790",
+      "email": ""
+  },
+  "record": {
+    "MX": [
+      "mx1.improvmx.com.",
+      "mx2.improvmx.com."
+    ],
```

remove the trailing dot
```suggestion
    "MX": [
      "mx1.improvmx.com",
      "mx2.improvmx.com"
    ],
```

---

oookkkayyyy ... i just copy pasted the records from improvmx